constants/communitySubject.go

```diff
@@ -1,22 +1,22 @@
 package constants
 
 const (
-	CommunityPostCreationSubject      = "community.post.new"
-	CommunityAsyncPostCreationSubject = "community.post.async.create"
-	CommunityAsyncPostUpdateSubject   = "community.post.async.update"
-	CommunityPostSchedulerSubject     = "community.post.scheduler"
+	CommunityPostCreationSubject      = "comt.post.new"
+	CommunityAsyncPostCreationSubject = "comt.post.async.create"
+	CommunityAsyncPostUpdateSubject   = "comt.post.async.update"
+	CommunityPostSchedulerSubject     = "comt.post.scheduler"
 
 	// CommunityNewReactionSubject is used for handling new reactions on feed objects
-	CommunityNewReactionSubject = "community.reactions.new"
+	CommunityNewReactionSubject = "comt.reactions.new"
 
-	CommunityPushNotificationSubject          = "community.pushNotifications.posts"    // subject for sending new post push notifications
-	CommunityCommentsPushNotificationsSubject = "community.pushNotifications.comments" // subject for sending new comment notification events
-	CommunityTagsPushNotificationsSubject     = "community.pushNotifications.tags"
+	CommunityPushNotificationSubject          = "comt.pushNotifications.posts"    // subject for sending new post push notifications
+	CommunityCommentsPushNotificationsSubject = "comt.pushNotifications.comments" // subject for sending new comment notification events
+	CommunityTagsPushNotificationsSubject     = "comt.pushNotifications.tags"
 
-	CommunityRealtimeMessagingSubject = "community.realtimeMessaging.feed" // subject for publishing real time feed
+	CommunityRealtimeMessagingSubject = "comt.realtimeMessaging.feed" // subject for publishing real time feed
 
-	CommunityNewCommentSubject          = "community.comment.new"
-	CommunityCommentUpvoteSubject       = "community.comments.upvotes.new"
-	CommunityCommentDownvoteSubject     = "community.comments.upvotes.delete"
-	CommunityCommentUpvoteRemoveSubject = "community.comments.upvotes.delete"
+	CommunityNewCommentSubject          = "comt.comment.new"
+	CommunityCommentUpvoteSubject       = "comt.comments.upvotes.new"
+	CommunityCommentDownvoteSubject     = "comt.comments.upvotes.del"
+	CommunityCommentUpvoteRemoveSubject = "comt.comments.upvotes.del"
```

Comment:
Let us stick with singular only going forward
```suggestion
	CommunityCommentUpvoteSubject       = "comt.comment.upvote.new"
	CommunityCommentDownvoteSubject     = "comt.comment.upvote.del"
	CommunityCommentUpvoteRemoveSubject = "comt.comment.upvote.del"
```